constants/index.ts

```diff
@@ -4,4 +4,11 @@ export const SUPPORTED_CHAINS = [
   'bchtest',
   'bchreg'
 ] // WIP integration tests to check if this corresponds to the database
```

Comment:
I believe we should leave WIP comments for _draft_ pull requests, not ones that are ready for review.